packages/react-docs/RELEASE-NOTES.md

```diff
@@ -90,6 +172,7 @@ releaseNoteTOC: true
 - Made APIs more react friendly with hooks ([#4342](https://github.com/patternfly/patternfly-react/pull/4342))
 - Allowed a promise to be returned by the create connector callback ([#4390](https://github.com/patternfly/patternfly-react/pull/4390))
 - Updated to sync with OpenShift Console ([#4317](https://github.com/patternfly/patternfly-react/pull/4317))
+- Created connector drag spec configuration ([#4529](https://github.com/patternfly/patternfly-react/pull/4529))
```

Comment:
This is from the 2020.09 release but it's under the 2020.08 section.

Reply:
@tlabaj it looks like this is still showing under 2020.08